base/src/main/java/io/spine/util/Preconditions2.java

```diff
@@ -37,31 +44,230 @@ private Preconditions2() {
     /**
      * Ensures that the passed string is not {@code null}, empty or blank string.
      *
-     * @param stringToCheck the string to check
+     * @param str
+     *         the string to check
      * @return the passed string
-     * @throws NullPointerException if the passed string is {@code null}
-     * @throws IllegalArgumentException if the string is empty or blank
+     * @throws IllegalArgumentException
+     *         if the string is empty or blank
+     * @throws NullPointerException
+     *         if the passed string is {@code null}
      */
     @CanIgnoreReturnValue
-    public static String checkNotEmptyOrBlank(String stringToCheck) {
-        return checkNotEmptyOrBlank(stringToCheck, "");
+    public static String checkNotEmptyOrBlank(String str) {
```

Comment:
Please re-use the `checkNotEmptyOrBlank` method with 2 args here as discussed vocally.